app/assets/stylesheets/pages/_machines.scss

```diff
@@ -0,0 +1,270 @@
+    max-width: 135px;
+    min-width: 52px;
+    width: 20%;
+  }
+
+  ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  li {
+    background-color: #f5f5f5;
+    cursor: pointer;
+    display: flex;
+    flex-basis: 33.333%;
+    justify-content: center;
+    text-align: center;
+    transition: all 0.3s ease-in-out;
+
+    &:hover,
+    &:focus {
+      background-color: lighten($c-pink, 5%);
+      text-decoration: underline;
+    }
+  }
+
+  li:not(.device-selected) {
+    opacity: 0.5;
+  }
+
+  a {
+    align-self: flex-end;
+    color: $c-text;
+    display: flex;
+    flex-basis: 100%;
+    flex-direction: column;
+    padding: rem(23) rem(0);
+    text-decoration: none;
+
+    @include breakpoint(min-width 1440px) {
+      flex-direction: row;
+      justify-content: center;
+
+      h2 {
+        margin-left: 4%;
+        margin-bottom: 3vh;
+        align-self: center;
+      }
+    }
+  }
+
+  h2 {
+    @include fluid-size("font-size", 14px, 36px);
+    margin-top: rem(22);
+  }
+
+  .device-selected {
+    background-color: $c-pink;
+    box-shadow: 3px 4px 19px rgba(0, 0, 0, 0.25);
+    position: relative;
+
+    a {
+      color: white;
+    }
+
+    &:after,
+    &:before {
+      border: solid transparent;
+      content: " ";
+      height: 0;
+      left: 50%;
+      pointer-events: none;
+      position: absolute;
+      top: 100%;
+      width: 0;
+    }
+
+    &:after {
+      border-top-color: $c-pink;
+      border-width: 15px;
+      margin-left: -15px;
+    }
+
+    &:before {
+      border-top-color: $c-pink;
+      border-width: 10px;
+      margin-left: -10px;
+    }
+  }
+}
+
+.device__problems {
+  @include clearfix();
+  padding: 1rem 0;
+
+  article {
+    border-bottom: 1px solid lighten($c-text, 50%);
+  }
+
+  header {
+    @include clearfix();
+    margin: 0 auto;
+    max-width: 600px;
+    padding: 1rem 0;
+    width: rd(355, 415);
+
+    &:hover,
+    &:focus {
+      h3 {
+        color: $c-pink;
+      }
+    }
+  }
+
+  header,
+  img {
+    cursor: pointer;
+  }
+
+  .__inactive-list-item {
+    border: 0;
+    clip: rect(0 0 0 0);
+    display: none;
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+  }
+
+  h3 {
+    @include fluid-size("font-size", 20px, 24px);
+    float: left;
+    font-family: $f-body;
+    font-weight: 400;
+    transform-origin: left;
+    transition: all 0.3s ease-in-out;
+  }
+
+  button {
+    background: none;
+    border: none;
+    float: right;
+    transition: all 0.3s ease;
+  }
+
+  .-flip-caret {
+    transform: rotate(180deg);
+  }
+
+  @include breakpoint(min-width 950px) {
+    ul {
+      @include clearfix();
+      margin: 0 auto;
+      max-width: 700px;
+      width: rd(682, 1024);
+    }
+
+    header {
+      max-width: 100%;
+      padding: 0 0 rem(24) 0;
+      width: 100%;
+    }
+
+    p {
+      margin: 0;
+      width: 100%;
+    }
+
+    li {
+      margin: 0 0 rem(48) 0;
```

Comment:
if there's no other margin, then just set `margin-bottom`. The short-hand is great when you're setting multiple margins, but is a little over-kill when you're only setting one side